app/handlers/trusted_parameters_launch.rb

```diff
@@ -0,0 +1,45 @@
+class TrustedParametersLaunch
+
+  attr_reader :user_state, :attrs
+
+  lev_handler
+
+  protected
+
+  def authorized?
+    true
+  end
+
+  def setup
+    @attrs = request.session['trusted']
+    @user_state = options[:user_state]
+  end
+
+  def handle
+
+    uuid_link = UserAlternativeUuid.find_by_uuid(attrs[:uuid])
+    if uuid_link.present?
+      outputs[:user] = uuid_link.user
+      user_state.sign_in!(outputs[:user])
+      return
+    end
+
+    if attrs['email']
+       user = LookupUsers.by_verified_email(attrs['email']).first
+       if user
+         UserAlternativeUuid.create(user: user, uuid: attrs['uuid'])
+         user_state.sign_in!(user)
+         outputs[:user] = user
+         return
+       end
+    end
```

Comment:
I think we still need the part of the flow where a user can log in to an existing account, right (where the trusted email doesn't match an existing email)? (in which case we would use the trusted parameters to add to their existing account and we'd also create a `UserAlternativeUuid` for them.